pipelines/treatment/bilhetagem/flows.py

```
@@ -44,18 +48,42 @@
+    + [
+        IntervalClock(
+            interval=timedelta(days=1),
+            start_date=datetime(
+                2022, 11, 30, 12, 0, tzinfo=timezone(smtr_constants.TIMEZONE.value)
+            ),
+            labels=[
+                smtr_constants.RJ_SMTR_AGENT_LABEL.value,
+            ],
+            parameter_defaults={"fallback_run": True},
+        )
+    ]
```

Ruff (0.13.1)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)